Add a README file for the sample problems (#178)
svalinn/ALARA: sample/readme.md

* with those materials and elements defined in the provided material and element
  libraries

The flux is read from the `hohlflux`, skipping the first 90 entries and scaling the values by 1e-10.

The irradiation schedule is more complex, with 
* a 1 day pulse repeated 25 times with 25 hours between pulses, followed by
* a 25 hour delay, followed by
* a 5 minute pulse repeated once

Results are produced at 5 cooling times for the waste 
disposal rating for NRC Class A and Class C waste, which
require the activity to be in units of Ci/m3.

## sample5

This advanced sample includes:
* a geometry with three intervals with different volumes
* each in a separate zone
* each of which has a mixture,
   * one with an element, 
   * one with an enriched element, and 
   * one with a material
* with those materials and elements defined in the provided material and element
  libraries

The flux is read from the `fluxin1`, and normalized differently for each interval.

The there is a single 10 year pulse.

Results are produced at 11 cooling times the total heat
and photon source, with units of Bq per cm3.

Most initial nuclides will have their chains truncated with a tolerance of 1e-6,

> but impurities (defined as nuclides that are less than 10 ppm) will be truncated

Reviewer: ```suggestion
but impurities (defined as nuclides with an initial concentration of less than 10 ppm) will be truncated
```
